x/lockup/keeper/lock.go

```diff
@@ -281,6 +281,40 @@ func (k Keeper) UnlockPeriodLockByID(ctx sdk.Context, LockID uint64) (*types.Per
 	return lock, err
 }
 
+// AddTokensToLock locks more tokens into a lockup
+func (k Keeper) AddTokensToLock(ctx sdk.Context, owner sdk.AccAddress, lockID uint64, coins sdk.Coins) (*types.PeriodLock, error) {
```

Comment:
Can we separate this out into two functions:

```
AddTokensToLockID(ctx sdk.Context, owner sdk.AccAddress, lockID uint64, coins sdk.Coins)
``` 
and 
```
AddTokensToLock(ctx, lock *types.PeriodLock, coins sdk.Coins)
```

I think both will come in handy

Reply:
Did split the function into two pieces